doc/api/wasi.md

```diff
@@ -22,15 +22,51 @@ const wasi = new WASI({
 const importObject = { wasi_snapshot_preview1: wasi.wasiImport };
 
 (async () => {
-  const wasm = await WebAssembly.compile(fs.readFileSync('./binary.wasm'));
+  const wasm = await WebAssembly.compile(fs.readFileSync('./demo.wasm'));
   const instance = await WebAssembly.instantiate(wasm, importObject);
 
   wasi.start(instance);
 })();
 ```
 
+To run the above example, create a new WebAssembly text format file i.e. `demo.wat`
+```wat
```

Comment:
Options here:

* Disable linter with a comment above this line until we can get `wat` added to the allowed-list:

```
<!--lint disable fenced-code-flag-->
```

...then re-enable it after the fenced code is closed (currently line 61)...

```
<!--lint enable fenced-code-flag-->
```

* Remove `wat` and go with `text` for now instead.

In either event, we can remove the workaround after `wat` is added to `remark-present-lint-node`.  If someone wants to do that now, check out https://github.com/nodejs/remark-preset-lint-node/tree/3779c21244b131f1f5d4293ea2ec6e6f1b67e5d7#add-new-language-or-grammar for instructions.